Data/Text.hs

```diff
@@ -1488,6 +1490,13 @@ chunksOf k = go
 -------------------------------------------------------------------------------
 -- ** Searching with a predicate
 
+-- | /O(n)/ The 'elem' function takes a character and a 'Text', and
+-- returns 'True' if the element is found in the given 'Text', or
+-- 'False' otherwise.
+elem :: Char -> Text -> Bool
+elem c t = isJust (find (== c) t)
```

Comment:
Why not define this in terms of the `S.any` primitive that `Data.Text.any` is based upon?

```
any :: (Char -> Bool) -> Text -> Bool
any p t = S.any p (stream t)
{-# INLINE any #-}
```

?

Reply:
Lack of knowledge of the `text` library is the reason. If that's the preferred way, let's do it.

Reply:
@emilypi let's do that then :-)